### DISCUSSION TITLE
How can i restore your backup plan?

### DISCUSSION THREAD

**Discussion by Nfo99** [olahallengren/sql-server-maintenance-solution#797] [General] [answered]
Good morning, I have implemented the backup plan using a full copy, a differential copy and the transaction log.

I am not clear how I should proceed to recover a backup of the last hour joining all the pieces together.... is there any manual, video or something that explains it clearly?

thank you very much

**Comment by azamia12** [comment, marked as answer]
You can use DBATools. IO from the PowerShell Gallery. It supports restoring backups made by this solution.
